GitUI/UserControls/RevisionGrid/RevisionGridControl.Command.cs

```diff
@@ -38,7 +38,8 @@ internal enum Command
             GoToMergeBaseCommit = 31,
             OpenCommitsWithDifftool = 32,
             ToggleBetweenArtificialAndHeadCommits = 33,
-            ShowReflogReferences = 34
+            ShowReflogReferences = 34,
+            HideStashBranch = 35,
```

Comment:
Positive, Stash is not branch

```suggestion
            ShowStashCommit = 35,
```

Reply:
Alternative - this is only one stash
```suggestion
            ShowMostRecentStash = 35,
```

Reply:
I have changed it to the suggested "ShowBranchReferences"